src/node_manager/http_client.rs

```diff
@@ -53,22 +24,21 @@ impl Response {
         self.0.json().await.map_err(Into::into)
     }
 
+    // TODO is this really needed ?
```

Comment:
I think it was required for quorum because  the responses didn't had `PartialEq, Eq` available, so we couldn't use them to compare the results, but should be possible now
The other places where it's used are now gone or also can be replaced with `.json()` I think, but needs to be checked

Reply:
Yeah I'll investigate this next 👍🏻